docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -852,10 +852,6 @@ btrim(str[, trim_str])
 
 #### Alternative Syntax
 
-```sql
-trim(BOTH trim_str FROM str)
-```
-
 ```sql
 trim(trim_str FROM str)
 ```
```

Comment:
@comphead  I found that `use_doc()` only supports one alternative syntax. Should we modify it to support multiple alternative syntaxes?

Reply:
oh its a vec, yeah we should do it, as well as simplify a little bit the `Documentation` structure to replace `Option<Vec>` to just `Vec`

Reply:
Lets leave trim not migrated for now and we can do it once the doc macros fixed

Reply:
- I filed https://github.com/apache/datafusion/issues/13944 to track

Reply:
I remove the change for `trim()`  in this PR, thanks!  I'd like to update `use_doc()` and migrate `trim()` simultaneously